.travis.yml

```diff
@@ -73,8 +17,20 @@ before_install:
 ## run
 # dnf install -y python ffmpeg portaudio libchromaprint
 
+install:
+  - pip3 install --upgrade pip setuptools wheel pbr
 
 script:
   - python3 setup.py build
-  - sudo python3 setup.py install
+  - pip3 install .
   - python3 -c "import sys; print(sys.path); import musicplayer as mp; p = mp.createPlayer()"
+
+# Deploy package to PyPi
+deploy:
+  provider: pypi
+  user: __token__
+  password : ${PYPI_TOKEN}
+  distributions: "sdist bdist_wheel"
+  skip_existing: true
+  on:
+    tags: true
```

Comment:
Also I used the `on tag` condition to publish the dists only when a tagged commit is built on travis but as pbr creates a dev version number for non tagged commits version we can also publish the package on each build on master.

For instance if the last tag was `0.0.1` and was three commits ago, then pbr will use the version number `0.0.2.dev3`, which will be registered on pypi as a pre-release version. [Pypi example](https://pypi.org/project/test-development-release/#history)